tutorial/tutorial-gabc-03.html

```diff
@@ -174,15 +199,19 @@
 <img src="kyrie-17-15.png"/>
 </div>
 
+<p>If you don't see the difference, look particularly at the initial "K" and note where the lower diaganol branches off from the upper one.</p>
+
 <p>The best-looking free font for gregorio is a matter of taste, but <code>kpfonts</code> is also a good choice (<code>\usepackage{kpfonts}</code>).</p>
 
+<p>For a list of free fonts which are readily available for <span class="tex">T<span class="epsilson">e</span>X</span> see <a href="http://www.tug.dk/FontCatalogue/">The <span class="latex">L<span class="alpha">a</span>T<span class="epsilon">e</span>X</span> Font Catalogue.</a>  Other fonts, including most of the ones already installed on your system, can also be used, but we'll leave it up to you to explore the use of other fonts on your own.</p>
```

Comment:
Maybe there could be an example on how to use a font with fontspec? Also, I don't think the LaTeX fonts are that interesting for Gregorian Chant... I think an example with 

```
\setmainfont{Arial}
```

or something like that will be quite bad for a gregorian chant font example, but quite good for this tutorial, as it will show how to load a system font, that should be available on many system... ?